docs/_docs/posts.md

```diff
@@ -155,6 +155,51 @@ you wish to access the currently-rendering page/posts's variables (the
 variables of the post/page that has the `for` loop in it), use the `page`
 variable instead.
 
+## Displaying post categories or tags
+
+Hey, that's pretty neat, but what about showing just some of your posts that are
+related to each other? For that you can use any of the [variables definable in
+Front Matter](https://jekyllrb.com/docs/frontmatter/). In the "typical post"
+section you can see how to define categories. Simply add the categories to your
+Front Matter as a [yaml
+list](https://en.wikipedia.org/wiki/YAML#Basic_components).
+
+Now that your posts have a category or multiple categories, you can make a page
+or a template displaying just the posts in those categories you specify. Here's
+a basic example of how to create a list of posts from a specific category.
+
+First, in the `_layouts` directory create a new file called `category.html` - in
+that file put (at least) the following:
+```html
+---
+layout: page
+---
+{% for post in site.categories.[page.category] %}
```

Comment:
I have never seen the `.[]` syntax before. Is that new?

I believe `site.categories[page.category]` will work here – please correct me if I'm wrong! This approach, if it's valid, will make this look a bit more like other programming languages, which can help folks new to Liquid understand what's going on here.

Reply:
It might be less "new" and more "what has worked for me."

I'll edit to be standardized.